components/explorer/src/lib.rs

```diff
@@ -372,6 +372,9 @@ impl<'a> IntoIterator for &'a BlockWithTransactions {
 /// | `location` | [`TxLocation`] | Location of the transaction in the block |
 /// | `location_proof` | [`ListProof`]`<`[`Hash`]`>` | Proof of transaction inclusion into a block |
 /// | `status` | (custom; see below) | Execution status |
+/// | `time` | [`DateTime`]`<`[`Utc`]`>` | An approximate commit time of the block which includes
```

Comment:
I'd suggest to use somewhat short description in the cell with a reference, which is explained below, like:

```markdown
| `time` | [`DateTime`]`<`[`Utc`]`>` | Block commitment time*

\* By commitment time we mean approximate commitment time for the block
which includes the transaction. This time is calculated as the arithmetic average
of the precommit local times of each validator.
```

IMHO with such an approach, the table will look nicer.

Reply:
Yeah. Looks nice. But question is How will it be looking on the docs.rs ? 🤔 Have you seen such approach before ?

Reply:
What do you mean? It will look just like in the markdown, e.g. 

![image](https://user-images.githubusercontent.com/12111581/72891721-f2835900-3d25-11ea-99a1-d0de9d9d45db.png)